### PR TITLE
Add 'Kabanero Contributor' badge info to site and CONTRIBUING.md 

### LINKED ISSUE #251: Adding Kabanero Contributor Badge and information onto both the main website and contribution website.
#### Feature description
Contribute to Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show such badge to your current/future employees that you have been recognized by your contribution to Kabanero, the open-source GitHub communities.  

https://www.youracclaim.com/org/ibm/badge/kabanero-contributor

#### Feature design or visual mockup
Kabanero_Contributor_Badge

#### Additional context

### PR DESCRIPTION
**Pull request by kSee04**
#### What was fixed?  (Issue # or description of fix)
Closes #251

This PR adds info & links about the `Kabanero Contributor` badge to the contribution page on the website and the CONTRIBUTING.md.

`SCREENSHOTS:`
Screen Shot 2019-10-16 at 10 53 46 AM
Screen Shot 2019-10-16 at 10 53 57 AM

### CODE REVIEW

File: src/main/content/robots.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 User-agent: *
-Disallow: /blog/redirected.html
-Noindex: /blog/redirected.html
```
Comment (kSee04) [inline src/main/content/robots.txt]: I don't know why it's saying that I changed this...here's a screenshot of this file in master currently
Screen Shot 2019-10-16 at 11 25 52 AM

Comment (alohr51) [reply src/main/content/robots.txt]: @kSee04 still need to update this out of the PR

Comment (alohr51) [reply src/main/content/robots.txt]: its talking about https://github.com/kabanero-io/kabanero-website/blob/master/src/main/content/robots.txt and your screen shot is the robots.txt on the root (there's 2 of them)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 Anyone can contribute to the Kabanero project and we welcome your contributions!
 
+Contribute to Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show such badge to your current/future employees that you have been recognized by your contribution to Kabanero, the open-source GitHub communities.
+
+https://www.youracclaim.com/org/ibm/badge/kabanero-contributor
+
+![Kabanero Badge](src/main/content/img/kabanero-badge.jpg)
```
Comment (alohr51) [inline CONTRIBUTING.md]: I would put this in its own sub header `## Earn the Kabanero Contribution badge`

Comment (alohr51) [reply CONTRIBUTING.md]: also update it to

>Contribute to the Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show this badge to anyone and prove that you have been recognized by your contribution to the open source Kabanero project.

File: src/main/content/_data/en/en.yml [hunk 3]
```diff
@@ -104,6 +104,10 @@ contribute:
     intro-title: Raising Issues
     intro-text: Please raise any bug reports on the Kabanero project repository's GitHub issue tracker. Be sure to search the list to see if your issue has already been raised. A good bug report is one that make it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
     open-issues:  Open issues for new features
+  badge:
+    title: Earning the Badge
+    text: Contribute to Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show such badge to your current/future employees that you have been recognized by your contribution to Kabanero, the open-source GitHub communities.
```
Comment (alohr51) [inline src/main/content/_data/en/en.yml]: update this to the same as above

**Review by alohr51 — CHANGES_REQUESTED** [review]

**Review by alohr51 — APPROVED** [review]

**Pull request merged**